wit/timezone.wit

```diff
@@ -3,53 +3,48 @@ package wasi:clocks@0.2.0;
 @unstable(feature = clocks-timezone)
 interface timezone {
     @unstable(feature = clocks-timezone)
-    use wall-clock.{datetime};
+    use system-clock.{instant};
 
-    /// Return information needed to display the given `datetime`. This includes
-    /// the UTC offset, the time zone name, and a flag indicating whether
-    /// daylight saving time is active.
+    /// Return information needed to display the given `instant`. This includes
```

Comment:
Maybe something like

```suggestion
    /// Return information needed to display the given `instant` in the user's configured time zone. This includes
```
?

Since `instant`s do not have associated time zone, it's confusing as currently worded: it doesn't give any indication of where the UTC offset or time zone name is coming from.

Reply:
Done.